### PR TITLE
alexander95015's Todoist Cheat Sheet

### PR DESCRIPTION
**Pull request by alexander95015**
# Goodie Cheat Sheet Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Provide a cheat sheet for Todoist users.

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://todoist.com/Help/KeyboardShortcuts

**Why did you choose this data source?**
It's the official support site for Todoist.

**Are there any other alternative (better) data sources?**
Probably not.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Todoist users.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No.

**Are you having any problems? Do you need our help with anything?**

**Where did you hear about DuckDuckHack? (For first time contributors)**
On the old DuckDuckGo site. (a few years back after the Snowden revelations)

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screen shot 2015-08-23 at 11 35 58 am

-----
IA Page: https://duck.co/ia/view/todoist_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@alexander95015 thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: share/goodie/cheat_sheets/json/todoist.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            "val":"Insert bold syntax"
+          }, {
+            "key":"[Ctrl] [i]",
+            "val":"Insert italics syntax"
+          }, {
+            "key":"[Shift] [Click]",
+            "val":"Complete and move to history a sub-task or a recurring task"
+          }],
+        "General shortcuts": [{
+            "key":"[Ctrl] [Click] or [Shift] [Click]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/todoist.json]: Let's break this one up into two pairs. The second should indicate it's an alternative shortcut.

**Comment by moollaza** [comment]
@alexander95015 this is looking pretty good. I only had one thing to comment on! You've done a great job :+1:

**Comment by alexander95015** [comment]
@moollaza thanks for the feedback. I've separated the two.

**Comment by moollaza** [comment]
LGTM :+1:

**Comment by moollaza** [comment]
@alexander95015 Thanks for taking the time to contribute! We really appreciate it.

We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@alexander95015 congrats! It's live: https://duckduckgo.com/?q=todoist+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!q

**Pull request merged**